internal/command/apps/releases.go

```diff
@@ -47,9 +47,23 @@ including type, when, success/fail and which user triggered the release.
 }
 
 func runReleases(ctx context.Context) error {
-	appName := app.NameFromContext(ctx)
+	var (
+		appName  = app.NameFromContext(ctx)
+		client   = client.FromContext(ctx).API()
+		releases []api.Release
+	)
+
+	app, err := client.GetAppCompact(ctx, appName)
+	if err != nil {
+		return fmt.Errorf("failed retrieving app %s: %w", appName, err)
+	}
+
+	if app.PlatformVersion == "machines" {
+		releases, err = client.GetAppReleasesMachines(ctx, appName, 25)
```

Comment:
please double check that the ordering is the same. this *might* return the first 25 releases instead of the most recent 25. we can fix it here or in web (probably makes sense to do in web if they are different).

Reply:
desired behavior is *most recent* 25, right? (this is what it does)

Reply:
yep, most recent 25 is desired behavior. awesome!